src/panda_eup/program_interpreter.py

```diff
@@ -427,9 +423,12 @@ def execute_apply_force_fingers(self, primitive_to_execute):
         return response.success
 
     # REVERT PRIMITIVE CALLBACK
-    def revert_user_sync(self, primitive_index):
+    def revert_user_sync(self, primitive_index, loadedProgram=True, program=None):
```

Comment:
Why was changing the function signature needed? it seems to me that loadedProgram should be a class variable, not a function argument. Maybe you want to do a function with the same name but different number of parameters (that inside calls the one with fewer)?